src/UnikraftExtension.ts

```diff
@@ -82,79 +94,69 @@ export class UnikraftExtension {
             }
         });
 
+    }
+
+    private initExtension() {
+        this.registerCommands();
         this.kraftStatusBarItem.text = 'Unikraft';
         vscode.window.showInformationMessage(
             'Congratulations, your extension "Unikraft" is now active!'
         );
     }
 
     private async setupExtension() {
-        var githubToken = utils.getGithubToken();
-    
-        if (githubToken === '') {
-            this.kraftStatusBarItem.text = 'Waiting for the Github Token...';
-            githubToken = await utils.updateGithubToken(this.kraftChannel);
-        }
-    
-        if (utils.getUkWorkdir() === '') {
-            utils.updateUkWorkdir(path.join(os.homedir(), '.unikraft'));
+        if (utils.getUkPackWorkdir() === '') {
+            utils.updateUkPackWorkdir(path.join(os.homedir(), '.local/share/kraftkit'));
         }
-    
-        if (!fs.existsSync(path.join(os.homedir(), '.kraftrc'))) {
-            this.installKraft(githubToken);
+
+        if (!fs.existsSync('/usr/local/bin/kraft')) {
+            this.installKraft();
+        } else {
+            this.initExtension();
         }
     }
 
-    private installKraft(githubToken: string) {
+    private installKraft() {
         const terminal = vscode.window.createTerminal(`Kraft setup`);
 
         this.kraftStatusBarItem.text = 'Installing kraft...';
-
-        this.kraftChannel.appendLine('Installing dependencies...');
-        terminal.sendText('sudo apt-get install -y --no-install-recommends \
-            build-essential libncurses-dev libyaml-dev flex git wget bison \
-            unzip uuid-runtime');
-        this.kraftChannel.appendLine('Installed dependencies.');
-
         this.kraftChannel.appendLine('Installing kraft...');
-        this.kraftChannel.appendLine(
-            execSync(
-                'pip install git+https://github.com/unikraft/kraft.git',
-                { env: env }
-            ).toString()
-        );
-        this.kraftChannel.appendLine('Installed kraft.');
-        this.kraftChannel.appendLine('Executing "kraft list update"...');
-        this.kraftChannel.appendLine(execSync(
-                'kraft list update',
-                {
-                    env: Object.assign(
-                        process.env,
-                        { 'UK_KRAFT_GITHUB_TOKEN': githubToken })
-                }
-            ).toString()
-        );
-        this.kraftChannel.appendLine('Executed "kraft list update".');
+        terminal.sendText(`sudo apt-get install -y --no-install-recommends build-essential \
+        libncurses-dev \
+        libyaml-dev \
+        flex \
+        wget \
+        socat \
+        bison \
+        unzip \
+        uuid-runtime \
+        python3 \
+        python3-setuptools \
+        python3-pip \
+        qemu-kvm \
+        qemu-system-x86 \
+        sgabios \
```

Comment:
I would also install qemu-system-arm

Reply:
Should these not be installed via the install script? I.e.:
```
curl --proto '=https' --tlsv1.2 -sSf https://get.kraftkit.sh | sh
```

Reply:
No @nderjung it's not available in the script https://get.kraftkit.sh.

Reply:
@craciunoiuc Done 👍🏽